smartseq2_single_sample/tasks/picard.wdl

```diff
@@ -17,7 +17,7 @@ task CollectMultipleMetrics {
       REFERENCE_SEQUENCE=${ref_genome_fasta} \
       ASSUME_SORTED=true
 
-      tar -cvf "${output_file}.tar" ${output_file}.*.txt
+      tar -cvf "${output_filename}.tar" ${output_filename}.*.txt
```

Comment:
the PR#50 will rewrite this section. basically, I am keeping individual files as outputs without tar them.

Reply:
Great. I look forward to changing it!